Source/SwiftLintFramework/Extensions/NSFileManager+SwiftLint.swift

```diff
@@ -9,11 +9,11 @@
 import Foundation
 
 public protocol LintableFileManager {
-    func filesToLintAtPath(_ path: String, rootDirectory: String?) -> [String]
+    func filesToLint(inPath: String, rootDirectory: String?) -> [String]
```

Comment:
See the "Compensate for weak type information" section of the guidelines:

![image](https://cloud.githubusercontent.com/assets/474794/21792266/56b952dc-d69d-11e6-812d-a2b5f425fecc.png)